types/KBaseStructure.ProteinStructures/display.yaml

```diff
@@ -0,0 +1,15 @@
+#
+# Define basic display information
+#
+name     : ProteinStructures
+subtitle : Lists of Protein Structures derived from pdb model and/or experimental files
```

Comment:
Is this a single list? Or multiple lists? It maybe should read:
`List of Protein Structures derived from PDB model and/or experimental files`